tao-macros/LICENSE-MIT

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020-2023 Ngo Iok Ui & Tauri Programme within The Commons Conservancy
```

Comment:
Should this be something like

> Copyright (c) 2017 - Present Tauri Apps Contributors

in https://github.com/tauri-apps/tauri/blob/dev/LICENSE_MIT ?

Reply:
guess so, but perhaps just 2022 - Present to match the crates lifetime?